README.md

```diff
@@ -1,24 +1,185 @@
-# README
+<a name="readme-top"></a>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+<div align="center">
 
-Things you may want to cover:
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
 
-* Ruby version
+  <h3><b>Microverse Education project by Matias Aguirre</b></h3>
 
-* System dependencies
+</div>
 
-* Configuration
+# 📗 Table of Contents
 
-* Database creation
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
 
-* Database initialization
+# 📖 Blog App <a name="about-project"></a>
 
-* How to run the test suite
+**Blog-App** is a fully functional website that shows the list of posts and empower users to interact with them by adding comments and liking posts.
 
-* Services (job queues, cache servers, search engines, etc.)
+## 🛠 Built With <a name="built-with"></a>
 
-* Deployment instructions
+### Tech Stack <a name="tech-stack"></a>
 
-* ...
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **Sign in**
+- **Adding posts for each user**
+- **Liking multiple posts**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://www.loom.com/share/74e8fa893b9b4fc2bc557193a20355d7)
+
+
```

Comment:
- [ **OPTIONAL** ] As you you are struggling about deploying your project online I can suggest try render bu using this **[guide](https://microverse.zendesk.com/hc/en-us/articles/10903290796179)** to help you doing so. If you deploy your project online I can suggest you add its link in your **README** so it can easily be accessed by users.

This issue is marked as **OPTIONAL** because I was warned by **Sr. CRS** to do so.